apps/aecore/src/aec_db.erl

```diff
@@ -0,0 +1,298 @@
+-module(aec_db).
+
+-export([check_db/0,           % called from setup hook
+         load_database/0,      % called in aecore app start phase
```

Comment:
Is this not called?

Reply:
... and I see `db_path` in deployment/ansible/deploy.yml not bumped. So it looks to me that the intention of this PR is not to restart chain (by not changing `db_path`) though, given that this `load_database` is not hooked anywhere (I understand), it will restart the chain - basically having `.chain` and `mnesia` side by side and completely independent. @uwiger Is that intentional?

Reply:
The `load_database()` function is called from a start_phase hook in `aecore`, which, unfortunately, has not been defined in `aecore.app`. I will fix that.

Reply:
I saw the fix and approved. So I understand intention is loading from a .chain old style DB if found.